run_test.py

```diff
@@ -62,7 +62,7 @@
         conf = {
             'base': 'ubuntu14_py2',
             'cuda': 'cuda65',
-            'cudnn': 'cudnn2',
+            'cudnn': 'cudnn5',
```

Comment:
You choose CuDNN v5.1 for Py3 and CuDNN v5 for example. Is this because you want to test the former combination?